packages/markdown/components/CodeTabs/index.jsx

```diff
@@ -7,8 +7,10 @@ const CodeTabs = props => {
   const { attributes, children } = props;
 
   function handleClick({ target }, index) {
-    const $wrap = target.closest('.CodeTabs');
-    $wrap.querySelectorAll('.CodeTabs_active').forEach(el => el.classList.remove('CodeTabs_active'));
+    console.log(target);
```

Comment:
```suggestion
```

Reply:
Oh word, thanks. And all of the emoji stuff will still work the same for the end user, but we could probably split the breaking syntax stuff out in to a separate script and fork the logic in IE, but between webpack config and IE debug, not sure it's worth the pain!